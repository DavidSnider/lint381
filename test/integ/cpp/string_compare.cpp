if (str.compare("foo") < 0) {

}

if (a.b) {
    compare()
}
