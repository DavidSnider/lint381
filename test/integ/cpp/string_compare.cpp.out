string_compare.cpp:1:8: error: Don't use 'string::compare'
if (str.compare("foo") < 0) {
       ^^^^^^^^
